br/br-checkpoint-backup.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 断点备份
 summary: 了解断点备份功能，包括它的使用场景、实现原理以及使用方法。
-aliases: ["/zh/tidb/dev/br-checkpoint"]
+aliases: ["/zh/tidb/stable/br-checkpoint"]
```

Comment:
I think this alias should be removed.

The url is existent: https://docs.pingcap.com/zh/tidb/stable/br-checkpoint

Reply:
`br/br-checkpoint.md` was renamed to `br/br-checkpoint-backup.md` in both master and release-7.1 branches.

Once `release-7.1` becomes the stable version, this URL https://docs.pingcap.com/zh/tidb/stable/br-checkpoint will become unavailable if this alias `/zh/tidb/stable/br-checkpoint` is removed.

Reply:
That makes sense. Thanks